src/index.ts

```diff
@@ -130,7 +134,25 @@ program
                    You can not rollback if you only have one released block.`
     )
     .action(() => {
-        rollback();
+        const versions = rollbackDetails();
+        const { current, name } = versions;
+        const message = `Do you want to rollback ${name} to the previous active release of version ${current}?`;
```

Comment:
Should we add a note about the impact of doing this?: "This will affect all the stores that have your block installed"

Reply:
Unless the CLI is intelligent enough to know when you're releasing a major version or not, it should not say this in my opinion. If you are releasing a major version, it won't affect any stores.